### PR TITLE
Docs building

### PR DESCRIPTION
**Pull request by ericfell**
skeleton for documentation

### CODE REVIEW

File: docs/source/examples/simulate_RFB_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,1151 @@
...
...JfLwU0OBQJAFJCYlr17bpMH5b1qkjT05eHpRoU53fe/ekEFlHaxR36MDOM2WKTYfJqdwTSeXQbPLSOXuWHeTqSuRIiiyjkSVZttE0lpAg+dU6nJIpGe4NUKyYDdeX6WDPkUiE2FjJPefkSasPi4+XxsoyZMGRFaHQ2YjRaKS5c+dSoUKFUhILFypUiObNm0dGGcvrqIUqF0RUlBRCZ++0Umps9BNJSJCmEmxJfZcRPAVYs2Y2HMSdtiz4adoMNysULGj105JPFcuhT3LatmXntDq44u5d+V7gRJLvlQ0aKi+J60CwtBmm0427dll5EHdWKlDAcVNpaKg0yLHSxBQRIWVKkeu2jIhgASaADTOnvI6xTTdSOnB3hlq1rD5EbuWeiMW58PQlVo85u3dnB3Tr5rgA3L3F2io8JMUg5Mkjn1XqxQvJAnz5spUH9ejhuLWWw91zvvzS6kP4bZkvn7pJhC0hFDoHiIqKoihbk5RpHFUuCK75lC4tXxVzXubGyuRA/GHl7y/PVAIRi4oEbKgDzd8cHh52es+n4tUr5msDsGRqmWA0SnqwnFFbixfbaHzlL/C33pJHAG56bdjQqt15dK7BYON0eSZwd75Ro6zY2WiUEqbaG2acmlq1bHLk43E1FSrId1sSETVpYoN1IzJSsqLIUUPbNMrTynuMz1bLpdxz+L1m1c/x4IE0yLHiXs4UPtgLCLBqsCdHLEZ6cFdZq4qZhIdLxbVtsKqlCw9Br1bNqt2TkqSMMVryneMIhU5ghuIXREKCZL9eskS+8/LpiezZM7VuOBowlRFlyrDzWlXDslMn+d8cHTuyc1qRp4mnfPPyYs9Nubh3T5ohybQGakyM5DFuTx5CS9y9a4MAUlUIGUteEpFUjaxMGSt2PnRI8nOSq6I9r1fVoUOmu758KQUIOxLtbYlZs9h5rUpXsXCh5EMo1wwI16R++inTXXlskcEgW9WwFLhfpVXXGd/ZykFJpphON1qRDWDrVsllQK7LkbNgATt3kyZW7DxtGtu5bl15Gn/8WMrZacX0LfeI8fW1PVOKEgiFzgqqV69Oz5Kv4mrVqlH16tXTXbI6il8Qa9dKpjE57demeacy8RPh6a28vR2uGJYGbmzq2TOTHR88kOaiZCqkTkTSFJMV1yYvlypLvc1U8NF9psEm3NerZEl5zUJcgJUrM9wtLEwyANhaFSI
```
Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:791]: `is only available for a symmetric cell`: do we currently enforce this?

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:799]: These equations aren't rendering for me

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:807]: This equation too

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:821]: `class` instead of `module` again

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:824]: The `eqref` renders to me as `(???)`. Note that I'm just looking at this in pycharm

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:929]: Maybe `(in the absolute value)` or `(in magnitude)`

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:929]: I would remove the `=` after the parameter names, it's sufficient to say that those are parameters.

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:929]: Typo in `current_discahrge`

Comment (jeremyfell) [inline docs/source/examples/simulate_RFB_example.ipynb:1003]: Nice 👍

File: docs/source/examples/simulate_RFB_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,1151 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fac22e99",
+   "metadata": {},
+   "source": [
+    "We'll first describe the redox flow cell and electrolytes to be cycled. A full description of all available parameters can be found in the documentation***insert link.\n",
+    "\n",
+    "Flow cell design is configured via the `redox_flow_cell.py` module. Adjustable, RFB-specific, parameters include electrode active and geometric area, cell ohmic resistance, initial concentrations of redox-active species (oxidized and/or reduced), and\n",
```
[the remark this answers is not in the document]
Comment (ericfell) [reply docs/source/examples/simulate_RFB_example.ipynb:45]: yes, switched to `electrode active area, electrode geometric area`

File: docs/source/examples/simulate_RFB_example.ipynb [hunk 3]
```diff
@@ -0,0 +1,1151 @@
...
+    "\n",
+    "2. **Symmetric Cell**: identical redox-actives in both reservoirs and a 0 V OCV when both reservoirs are at 50% state-of-charge (SOC)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "073e9afe",
+   "metadata": {},
+   "source": [
+    "Below we define a cell with a 5 mL CLS, 10 mL NCLS, and 10 mM initial concentrations for all four species ($C_{ox,\\scriptscriptstyle CLS},~C_{red,\\scriptscriptstyle CLS},~C_{ox,\\scriptscriptstyle NCLS},~C_{red,\\scriptscriptstyle NCLS}$). The voltage difference between the formal reduction potential ($E^{o^{\\prime}}$) of the posolyte and negolyte species is 1.0 volts, defined as $E_{cell}^{o^{\\prime}}=E_{\\scriptscriptstyle posolyte}^{o^{\\prime}}-E_{\\scriptscriptstyle negolyte}^{o^{\\prime}}$. \n",
```
[the remark this answers is not in the document]
Comment (ericfell) [reply docs/source/examples/simulate_RFB_example.ipynb:58]: done

File: docs/source/examples/simulate_RFB_example.ipynb [hunk 4]
```diff
@@ -0,0 +1,1151 @@
...
+    "A cell ohmic resistance of 1.0 $\\Omega$, and electrochemical rate constants ($k_{o}$) of $10^{-3}$ cm/s for both posolyte and negolyte species, is also defined. The `ZeroDModel` class defaults to one-electron transfers for redox-active species but this can be changed, as we'll show later on."
```
[the remark this answers is not in the document]
Comment (ericfell) [reply docs/source/examples/simulate_RFB_example.ipynb:59]: done

File: docs/source/examples/simulate_RFB_example.ipynb [hunk 5]
```diff
@@ -0,0 +1,1151 @@
...
+   "source": [
+    "Cells can be electrochemically cycled by constant current (**CC**), constant current followed by constant voltage (**CCCV**), or constant voltage (**CV**) protocols. CC and CCCV cycling require user input of applied currents, while CCCV and CV cycling require input of current cutoffs for charge and discharge. All techniques require input of voltage limits for charge and discharge. If at any time during CCCV cycling the cell cannot provide the desired applied current, CV cycling will take place."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c328337f",
+   "metadata": {},
+   "source": [
+    "We define a CC protocol with the `ConstantCurrent` module to charge the flow cell up to 1.4 V and discharge down to 0.6 V, both at a current of 100 mA."
```
[the remark this answers is not in the document]
Comment (ericfell) [reply docs/source/examples/simulate_RFB_example.ipynb:104]: done

File: docs/source/examples/simulate_RFB_example.ipynb [hunk 6]
```diff
@@ -0,0 +1,1151 @@
...
+    "## 4a. Plot cycling experiment with Matplotlib \n",
+    "#### (or plotting package of your choice)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bf0d44a",
+   "metadata": {},
+   "source": [
+    "The `results` tuple obtained from our simple RFB cycling simulation has multiple time profiles of useful cell information such as voltage, current, capacities, concentrations, overpotentials etc. Let's plot a few outputs describing the cell we just ran:"
```
[the remark this answers is not in the document]
Comment (ericfell) [reply docs/source/examples/simulate_RFB_example.ipynb:173]: changed to : The `results` object is obtained from our simple RFB cycling simulation and is an instance of the `CyclingProtocolResults` class. This object has multiple...

File: docs/source/examples/simulate_RFB_example.ipynb [hunk 7]
```diff
@@ -0,0 +1,1151 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61303936",
+   "metadata": {},
+   "source": [
+    "Optional capacity fade mechanisms can also be incorporated. These include chemical degradation, chemical redox of active species (e.g. self-discharge), dimerization, or multiple stacked degradation mechanisms. Rate constants and reaction rate orders can be adapted as needed to the electrolyte chemistries in each reservoir. A mechanism class is first declared, and then called within a cycling protocol's `.run()` method.\n",
+    "<br>\n",
+    "The same mechanism can be applied to both CLS and NCLS via `.run(degradation=...)` (standard for a symmetric cell), or individual reservoirs can be selected for degradation mechanisms e.g., `.run(cls_degradation=..., ncls_degradation=...)`"
```
[the remark this answers is not in the document]
Comment (ericfell) [reply docs/source/examples/simulate_RFB_example.ipynb:521]: no

File: docs/source/index.rst [hunk 8]
```diff
@@ -3,14 +3,69 @@
...
+-   NumPy
+
+
+Example Jupyter notebooks are provided in the examples/ directory.
+
+Examples and Documentation
+---------------------------
+
+The documentation can be found at
+`rfbzero.readthedocs.io <https://ericfell-rfbzero.readthedocs.io/en/latest/>`_.
```
[the remark this answers is not in the document]
Comment (ericfell) [reply docs/source/index.rst:54]: yes because readthedocs makes a homepage, and this is the index page

**Review by jeremyfell — APPROVED** [review]
LGTM

File: docs/source/examples/simulate_RFB_example.ipynb (hunk 1 above)
Comment (ericfell) [reply docs/source/examples/simulate_RFB_example.ipynb]: fixed by #21

**Pull request merged**